test/boost/sstable_compaction_test.cc

```diff
@@ -2830,6 +2830,8 @@ SEASTAR_THREAD_TEST_CASE(sstable_scrub_reader_test) {
 
 SEASTAR_TEST_CASE(sstable_run_based_compaction_test) {
     return test_env::do_with_async([] (test_env& env) {
+        env.set_use_uuid_sstable_identifiers(false);
```

Comment:
It's better to use test_env_config to create the test_env with the desired value of use_uuid instead of a setter function